src/test/java/seedu/address/logic/parser/consultation/ExportConsultCommandParserTest.java

```diff
@@ -0,0 +1,91 @@
+package seedu.address.logic.parser.consultation;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+import seedu.address.logic.commands.consultation.ExportConsultCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+public class ExportConsultCommandParserTest {
+    @TempDir
+    public Path temporaryFolder;
+
+    private ExportConsultCommandParser parser;
+    private Path dataDir;
+
+    @BeforeEach
+    public void setUp() throws IOException {
+        parser = new ExportConsultCommandParser();
+        dataDir = temporaryFolder.resolve("data");
+        Files.createDirectories(dataDir);
+    }
+
+    @AfterEach
+    public void tearDown() throws IOException {
```

Comment:
I hope this doesn't get called out for being arrowhead code, but just in case, maybe consider extracting the try catch block inside the lambda to be a separate private function? Would help with SLAP.